clients/stellar-relay-lib/src/connection/connector/message_handler.rs

```diff
@@ -80,29 +78,23 @@ impl Connector {
 				} else {
 					self.send_auth_message().await?;
 				}
-				log::info!("process_stellar_message(): Hello message processed successfully");
+				info!("process_stellar_message(): Hello message processed successfully");
 			},
 
 			StellarMessage::Auth(_) => {
 				self.process_auth_message().await?;
 			},
 
 			StellarMessage::ErrorMsg(e) => {
-				log::error!(
-					"process_stellar_message(): Received ErrorMsg during authentication: {e:?}"
-				);
+				error!("process_stellar_message(): Received ErrorMsg during authentication: {e:?}");
 				if e.code == ErrorCode::ErrConf || e.code == ErrorCode::ErrAuth {
 					return Err(Error::from(e))
 				}
 				return Ok(Some(StellarMessage::ErrorMsg(e)))
 			},
 
+			// we do not handle other messages. Return to caller
 			other => {
-				log::trace!(
```

Comment:
we are only interested on messages that will be processed by the vault.
Logging should be done over there than here.